R/wrappers.R

```diff
@@ -432,12 +448,15 @@ prop_test <- function(x, formula,
                       conf_int = TRUE,
                       conf_level = 0.95,
                       success = NULL,
+                      correct = TRUE,
```

Comment:
I believe that the best practice is having `NULL` as default for argument which default value depends on something. It seems a little bit confusing having statement `correct = TRUE` with details in documentation. In case of `correct = NULL` it motivates user to read documentation.